src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -86,6 +88,20 @@ const std::vector<DataView::Column>& LiveFunctionsDataView::GetColumns() {
   return columns;
 }
 
+[[nodiscard]] static std::string BuildTypeColumnsString(
```

Comment:
I find the name a bit confusing because this builds only part of the string shown in the "Type" column.

Reply:
`BuildTypeString` ?

Reply:
Better. A more verbose suggestion: `BuildTypePartOfTypeColumnString`. The problem is that "type" is still overloaded as the column is called "Type" but contains more info.
Would changing the name of the column also be an option? Like "Type & Hooked" or "Type [Hooked]".

Reply:
> Would changing the name of the column also be an option? Like "Type & Hooked" or "Type [Hooked]".
The column names already don't fit horizontally.